minio/api.py

```diff
@@ -1045,14 +1045,33 @@ def stat_object(self, bucket_name, object_name):
                                   object_name=object_name)
 
         etag = response.headers.get('etag', '').replace('"', '')
+        response.headers.pop('etag');
```

Comment:
Actually the idea was earlier to just save the response.headers verbatim. But since we are filtering now its not needed.